config/docker/dependencies/ubuntu/openmpi/Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN export DEBIAN_FRONTEND=noninteractive &&\
                     git \
                     libopenmpi-dev \
                     libhdf5-openmpi-dev \
+                    libhdf5-serial-dev \
```

Comment:
The serial one is libhdf5-dev

Reply:
I like the clarity of the serial nature explicitly in the name and think it should stay. Any reason not to use this one?

Reply:
I was aware of the existence of libhdf5-serial-dev as I look at packages at synaptic.
If I use apt, it exists.
```
Note, selecting 'libhdf5-dev' instead of 'libhdf5-serial-dev'
```
So let us keep 'serial'